obs_scripts/rsky.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+import time
+from _observation import Observation
+
+
+class RSky(Observation):
+    """
+    An observing module for R-SKY observation, which provides Tsys
+    measurement toward (az, el)=(45, 70))
+    """
+
+    def __init__(self):
+
+        ObservationType = "R-SKY"
+        super().__init__()
```

Comment:
This is not an instance variable nor class variable.
Please define `ObservationType` **outside of** `__init__` scope like the following snippet.

```python
class RSky(Observation):
    """docstring..."""

    ObservationType = "R-Sky"

    def run(self, integ_time):
        ...
```

And once again, in this case, there's no need to define `__init__` method, because if there's no `__init__` method in the class, the parent class's will be called, i.e. the following script can completely be dropped:

```python
    def __init__(self, obs_file=None):
        super().__init__(obs_file)
```

If you want to reject any argument, you can define `__init__` method with such comment.